src/js_crypto/constants.ts

```diff
@@ -0,0 +1,328 @@
+// @gen this file is generated - don't edit it directly
+export { prefixes, versionBytes, poseidonParams }
+
+let prefixes = {
```

Comment:
To avoid human error, I'd suggest automatically padding these and asserting the correct length.

Reply:
these are auto-generated from OCaml, and I think we avoid human error there :)

Reply:
Cool 👍🏼